sdk/ml/azure-ai-ml/pyproject.toml

```diff
@@ -6,6 +6,8 @@ pylint = true
 mindependency = false
 latestdependency = false
 black = true
+absolute_cov = true
+absolute_cov_percent = 22.40
```

Comment:
This is the disagreeing low number. I'm going to keep this around and check it in so we can prevent further negative coverage movement. @pdhotems I've filed another issue #39878 to do a deeper dive into where the `coverage` overall percent is getting changed. In the meantime we can prevent any further backslide in your package while I continue to investigate.